packages/bp/src/core/realtime/realtime-service.ts

```diff
@@ -134,8 +134,17 @@ export class RealtimeService {
 
     if (this.useRedis) {
       const redisFactory = this.monitoringService.getRedisFactory()
-
-      if (redisFactory) {
+      if (redisFactory('commands')) {
```

Comment:
Not calling `redisFactory` would make the condition always return true since it always returns a function. When using the `pro-monitoring-service`, it returns a Redis client, but the `monitoring-service` returns a function that returns undefined.